CVP-1871 Enable more checks for ansible linting in upstream playbooks

LikhithaEda opened the pull request:
**CVP-1871**
The changes I made are getting rid of ansible-lint warnings & errors. 
Changes were made in all yml files.
I resolved errors such as : 
   - too many spaces before colon  (colons)
   - too few spaces before comment  (comments)
   - line too long (98 > 80 characters)

samvarankashyap commented on ansible-lint.yml:
```diff
@@ -24,4 +24,4 @@ skip_list:
   - '601'
-  - '602'
\ No newline at end of file
+  - '602'
```
@LikhithaEda Why is 602 added. 
which rule is it.

14rcole reviewed (approved):
LGTM nice work

samvarankashyap commented:
@LikhithaEda 
Please change the title of the PR to following format

CVP-1871 <title of the ticket >

LikhithaEda reviewed (commented):
> @LikhithaEda
> Please change the title of the PR to following format
> 
> CVP-1871 <title of the ticket >

done

lslebodn commented:
@LikhithaEda  Please rebase on top of current master

lslebodn commented:
BTW @mvalarh introduced some config file as well as part of his change
https://github.com/redhat-operator-ecosystem/operator-test-playbooks/blob/master/.ansible-lint

It would be good to use just a one file

lslebodn reviewed (changes requested):
https://github.com/redhat-operator-ecosystem/operator-test-playbooks/pull/157/checks?check_run_id=2892725111

The check  "Run ansible-lint and yamllint for this repo. / build" still do not pass

-1

lslebodn commented:
@LikhithaEda  Could you explain why new file local.yml was added in the pull request.
We were supposed to fix linting issues. I cannot see a relation with adding new files.

Moreover, I can still se two configuration for ansible lint in different files.
* https://github.com/LikhithaEda/operator-test-playbooks/blob/local/.ansible-lint
* https://github.com/LikhithaEda/operator-test-playbooks/blob/local/ansible-lint.yml

We should use just one configuration.

lslebodn reviewed (changes requested):
Some of question still has not been answered ^^

LikhithaEda reviewed (commented):
> https://github.com/redhat-operator-ecosystem/operator-test-playbooks/pull/157/checks?check_run_id=2892725111
> 
> The check "Run ansible-lint and yamllint for this repo. / build" still do not pass
> 
> -1

I solved this issue in the new commit

LikhithaEda commented:
> @LikhithaEda Could you explain why new file local.yml was added in the pull request.
> We were supposed to fix linting issues. I cannot see a relation with adding new files.
> 
> Moreover, I can still se two configuration for ansible lint in different files.
> 
>     * https://github.com/LikhithaEda/operator-test-playbooks/blob/local/.ansible-lint
> 
>     * https://github.com/LikhithaEda/operator-test-playbooks/blob/local/ansible-lint.yml
> 
> 
> We should use just one configuration.

I did not add the local.yml file. It was already there and just modified to fix linting errors. I think after the rebase it is showing as a new file. And the reason the check failed previously is because of a syntax error in the local.yml which I fixed this time.

LikhithaEda commented:
> BTW @mvalarh introduced some config file as well as part of his change
> https://github.com/redhat-operator-ecosystem/operator-test-playbooks/blob/master/.ansible-lint
> 
> It would be good to use just a one file
I rebased after this changes were merged but I don't see this file locally for some reason. Should I add it myself?

lslebodn commented:
> @lslebodn I did not add the local.yml file. It was already there and just modified to fix linting errors. I think after the rebase it is showing as a new file. And the reason the check failed previously is because of a syntax error in the local.yml which I fixed this time.

Github says you added the file.
* https://patch-diff.githubusercontent.com/raw/redhat-operator-ecosystem/operator-test-playbooks/pull/157.patch
```
diff --git a/local.yml b/local.yml
new file mode 100644
index 00000000..f93dc3cf
--- /dev/null
+++ b/local.yml
@@ -0,0 +1,112 @@
+---
+- name: Test
```

lslebodn commented:
@samvarankashyap  please help @LikhithaEda  with rebase

lslebodn reviewed (changes requested):
The story says:
```
Remove as much rules from the ansible-lint.yml skip_list as possible
```
and there are also list of rules which should be removed from the configuration file.
But we did not change anything in configuration.
There is just new-line added.

Fixing linting violation is nice but we need to ensure nobody else will introduce them in the future. Which is not done in current MR.

We should also remove duplicate configuration for ansible lint
* https://github.com/LikhithaEda/operator-test-playbooks/blob/local/.ansible-lint
* https://github.com/LikhithaEda/operator-test-playbooks/blob/local/ansible-lint.yml

There is not any reason to have two different files for the same purpose.

LikhithaEda commented:
> The story says:
> 
> ```
> Remove as much rules from the ansible-lint.yml skip_list as possible
> ```
> 
> and there are also list of rules which should be removed from the configuration file.
> But we did not change anything in configuration.
> There is just new-line added.
> 
> Fixing linting violation is nice but we need to ensure nobody else will introduce them in the future. Which is not done in current MR.
> 
> We should also remove duplicate configuration for ansible lint
> 
>     * https://github.com/LikhithaEda/operator-test-playbooks/blob/local/.ansible-lint
> 
>     * https://github.com/LikhithaEda/operator-test-playbooks/blob/local/ansible-lint.yml
> 
> 
> There is not any reason to have two different files for the same purpose.

The two files: **.ansible-lint** and **ansible-lint.yml** have different contents.

wheelerlaw commented:
@LikhithaEda and I merged the `.ansible-lint` and `ansible-lint.yml` file into a single `.ansible-lint` file.

dirgim commented:
/retest

wheelerlaw commented:
> /retest

Lihkitha will need to rebase on top of the current master in order for the checks to pass.

14rcole commented on .ansible-lint:
```diff
@@ -17,4 +16,14 @@ skip_list:
   # - yaml  # Violations reported by yamllint
   - line-length
-  - comments
+  - '102'
```
Can you add comments to these lines saying what each of the errors is

14rcole commented:
LGTM

wheelerlaw commented:
lgtm. let's get this merged in for now and we can make more changes in the future.

lslebodn commented:
/test CVP
